core/templates/dev/head/pages/exploration_editor/editor_tab/add_answer_group_modal_directive.html

```diff
@@ -43,6 +43,11 @@ <h3>Add Response</h3>
                                 add-state="addState">
     </outcome-destination-editor>
   </form>
+
+  <div class="oppia-correctness-label-editor" ng-if="isCorrectnessFeedbackEnabled() && !isCurrentInteractionLinear()" style="margin-left: 4px; margin-top: 18px;">
```

Comment:
Do we have used this class `oppia-correctness-label-editor` for styling or anything else?

Also, can we remove the inline styling?

Reply:
Done, removed inline styling, added it to `oppia-correctness-label-editor` class.